Form/Type/CategorySelectorType.php

```diff
@@ -40,10 +41,17 @@ public function __construct(ManagerInterface $manager)
     }
 
     /**
+     * NEXT_MAJOR: Remove method, when bumping requirements to SF 2.7+.
+     *
      * {@inheritdoc}
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        $this->configureOptions($resolver);
```

Comment:
We don't do it in that case for other bundles, at least not on the methods I found, maybe because the only consumer of this method is Symfony? Or maybe it already throws a deprecation? Or because we don't want people in 2.3 (for instance) to get a deprecation notice for that?